From 42409fda665293dadf442df4d186c7f38ba83a05 Mon Sep 17 00:00:00 2001
From: Olivier Brunel <jjk@jjacky.com>
Date: Sat, 14 Jan 2017 18:20:33 +0100
Subject: [PATCH 6/7] plugins/notification: Update more tray's items
 sensitivity

Only the "Get Mail" menu item of the tray's popup was disabled when
Claws is locked, but the per-account get mail & Exit also should be.

Signed-off-by: Olivier Brunel <jjk@jjacky.com>
---
 src/plugins/notification/notification_trayicon.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/plugins/notification/notification_trayicon.c b/src/plugins/notification/notification_trayicon.c
index 720183722..fff19e72d 100644
--- a/src/plugins/notification/notification_trayicon.c
+++ b/src/plugins/notification/notification_trayicon.c
@@ -469,6 +469,8 @@ static void notification_trayicon_on_popup_menu(GtkStatusIcon *status_icon,
 	cm_toggle_menu_set_active("SysTrayiconPopup/ShowBubbles", notify_config.trayicon_popup_enabled);
 #endif
 	cm_menu_set_sensitive("SysTrayiconPopup/GetMail", mainwin->lock_count == 0);
+	cm_menu_set_sensitive("SysTrayiconPopup/GetMailAcc", mainwin->lock_count == 0);
+	cm_menu_set_sensitive("SysTrayiconPopup/Exit", mainwin->lock_count == 0);
 
   updating_menu = FALSE;
 
-- 
2.11.0

